app/templates/view_projects.html

```diff
@@ -0,0 +1,67 @@
+{% extends "layout.html" %}
+
+{% block title %} View Projects {% endblock %}
+
+{% block scripts %}
+<script type="text/javascript" src="{{ url_for('static', filename='scripts/table_sort.js') }}"></script>
+<script type="text/javascript" src="{{ url_for('static', filename='scripts/clear_form.js') }}"></script>
+
+<script>
+window.onload = function()
+{
+    // Make the table sortable and sort it by most recent
+    var timesTable = document.body.getElementsByTagName("table")[0];
+    makeSortable(timesTable);
+    sortTable(timesTable, 3, -1);
+}
+</script>
+{% endblock %}
+
+{% block body %}
+    <div class="filter-params">
+        <form method="POST">
+            {{ form.csrf_token }}
+            <p>{{ form.username.label }} {{ form.username }}</p>
+            <p>{{ form.slug.label }} {{ form.slug }}</p>
+            <p>{{ form.include_deleted.label }} {{ form.include_deleted }}</p>
+            <p>{{ form.include_revisions.label }} {{ form.include_revisions }}</p>
+            <p><input type="submit" value="Submit">  <input type="button" name="clear" value="Clear" onclick="clearForm(this.form, false);"></p>
+        </form>
+    </div>
+        <div class="view-projects">
+        <table>
+            <thead>
+                <tr>
+                    <th class="alpha">Name</th>
+                    <th class="list">Slugs</th>
+                    <th class="list">Members</th>
+                    <th class="list">Managers</th>
+                    <th class="list">Spectators</th>
+                    <th>URI</th>
+                </tr>
+            </thead>
+
+            {% for p in projects %}
+            <tr>
+                <td>{{ p['name'] }}</td>
+                <td>{{ p['slugs'] | join(' ') }}</td>
+                <td>{{ p['members'] | join(' ') }}</td>
+                <td>{{ p['managers'] | join(' ') }}</td>
+                <td>{{ p['spectators'] | join(' ') }}</td>
+                <td>{{ p['uri'] }}</td>
+                {% if admin %}
+                    <td><a href="{{ url_for('edit_project', project=p['slugs'][0]) }}">EDIT</a></td>
+                {% else %}
+                    {% for manager in p['managers'] %}
```

Comment:
If you view the page as a project manager that isn't an admin, this will create a lot of unnecessary empty `<td>`s

[Example](http://imgur.com/h7cey4J) (logged in as "test" with password "test" on the staging server)